pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -297,6 +331,20 @@ public boolean isIgnoreCase() {
     }
   }
 
+  public static class TableNamesExtractionRequest {
+    private final List<String> _sqls;
```

Comment:
Why does the request need to be a list of SQL queries? Can't we loop on the client side instead?

Reply:
In any case, I think we probably don't need this class and can use the string list directly?